Resources/Prototypes/Catalog/Jukebox/Standard.yml

```
@@ -39,3 +39,71 @@
+
+- type: jukebox
+  id: Spaced
+  name:  pinkblossom6 - Spaced
+  path:
+    path: /Audio/_White/Lobby/spaced.ogg
+
+- type: jukebox
+  id: SpaceOddity
+  name: David Bowie - Space Oddity
+  path:
+    path: /Audio/_White/Lobby/space-oddity.ogg
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)